File: sem1/python/prac-6.py
#6.1)write a program to find the sum of first number...

# n = int(input("Enter the number: "))
# sum = 0
# for i in range(1,n+1):
#   sum = sum + i
# print(f"Sum of first number: {sum}")

#6.2)write a program to generate fibonacci series from 1 to 15...

# a = 0
# b = 1
# print(a)
# print(b)
# for i in range(1,15):
#   c = a + b
#   print(c)
#   a = b
#   b = c

#6.3)write a program to print the multiplication table from 2 to 10...

for i in range(2,11):
  print("Table of ",i)
  for j in range(1,11):
    print(i,"x",j,"=",i*j)

File: sem1/C/prac-6.c
//6.a.Write a program to print the memory address using pointers.

#include <stdio.h>
int main()
{
   int x;
   int *p;
   
   printf("\nEnter a number: ");
   scanf("%d",&x);
   p = &x;
   printf("\nValue: %d",*p);
   printf("\nAddress: %u",p);
   
    return 0;
}

//6.b.Write a program for multiplication of two pointers.

#include <stdio.h>
int main()
{
    int x,y,mul;
    int *p1;
    int *p2;
   
    printf("\nEnter 1st number: ");
    scanf("%d",&x);
    printf("\nEnter 2nd number: ");
    scanf("%d",&y);
    p1 = &x;
    p2 = &y;
   
    mul = (*p1) * (*p2);
    
    printf("\n%d X %d = %d",*p1,*p2,mul);
   
    return 0;
}

//6.c.Write a program to print the Index,Number,and Address of 5 numbers.

#include <stdio.h>

int main() {
    int arr[5];
    int *p;
    
    // Input values into the array
    for (int i = 0; i < 5; i++) {
        printf("Enter number %d: ", i + 1);
        scanf("%d", &arr[i]);
    }
    
    p = &arr[0];
    
    printf("\n**************************************");
    printf("\nIndex\tNumber\tAddress");
    

    for (int i = 0; i < 5; i++) {
        printf("\n%d\t%d\t%u", i + 1, *p, p);
        p++; 
    }
    
    return 0;
}
